tests/Doctrine/DBAL/Migrations/Tests/Functional/FunctionalTest.php

```diff
@@ -430,8 +431,8 @@ public function testSkippedMigrationsDispatchesTheExpectedEvents()
      */
     public function testMigrateWithConnectionWithAutoCommitOffStillPersistsChanges()
     {
-        $listener            = new AutoCommitListener();
-        list($conn, $config) = self::fileConnectionAndConfig();
```

Comment:
@carusogabriel We may want to add sniff for this.

Reply:
What is these? I didn’t get it

Reply:
Short list syntax: `list($foo, $bar) = $someArray;` can be written as `[$foo, $bar] = $someArray;`.